src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -172,4 +172,20 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
+
+## Reviewing quickstart descriptions
+
+External vendors write quickstart descriptions. When one of these are ready for a review, they open an issue in our repo with a link out to the draft. These are written with SEO in mind, so they will appear differnt than most of our other docs and will not fully follow our style guide. 
+
+To close these issues:
+
+1. Follow the link to the draft.
+2. Review the draft with the tips below. 
+3. Leave comments or direct suggestions.
+4. Close the issue and leave a comment saying the review is done.
+
+Here are a few helpful tips:
+- These writers are not familiar with New Relic. Ensure they use our products correctly in the descriptions.
+- They don't know all the nuances of our style guide. It is shared with them, but ensure they follow confusing aspects of it.
```

Comment:
@bradleycamacho how about a slightly different tone like this: We shared the style guide with them, but they may need help following it.